packages/kbn-scout/src/playwright/fixtures/test/synthtrace/index.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import {
+  ApmSynthtraceEsClient,
+  OtelSynthtraceEsClient,
+  createLogger,
+  LogLevel,
+  ApmSynthtraceKibanaClient,
+  InfraSynthtraceEsClient,
+  InfraSynthtraceKibanaClient,
+} from '@kbn/apm-synthtrace';
+import { Readable } from 'stream';
+import type {
+  ApmFields,
+  Fields,
+  InfraDocument,
+  OtelDocument,
+  Serializable,
+  SynthtraceGenerator,
+} from '@kbn/apm-synthtrace-client';
+import Url from 'url';
+import { coreWorkerFixtures } from '../../worker';
+
+type SynthtraceEvents<T extends Fields> = SynthtraceGenerator<T> | Array<Serializable<T>>;
+
+export interface SynthtraceFixture {
+  apmSynthtraceEsClient: {
+    index: (events: SynthtraceEvents<ApmFields>) => Promise<void>;
+    clean: ApmSynthtraceEsClient['clean'];
+  };
+  infraSynthtraceEsClient: {
+    index: (events: SynthtraceEvents<InfraDocument>) => Promise<void>;
+    clean: InfraSynthtraceEsClient['clean'];
+  };
+  otelSynthtraceEsClient: {
+    index: (events: SynthtraceEvents<OtelDocument>) => Promise<void>;
+    clean: OtelSynthtraceEsClient['clean'];
+  };
+}
+
+export const synthtraceFixture = coreWorkerFixtures.extend<SynthtraceFixture>({
+  apmSynthtraceEsClient: async ({ esClient, config, kbnUrl }, use) => {
+    const logger = createLogger(LogLevel.info);
```

Comment:
small question here:
- can we use already existing `log` instance from `coreFixtures` for consistency with other client (ES, Kibana)?